src/test/java/ginger/parser/InputParserTest.java

```diff
@@ -0,0 +1,77 @@
+package ginger.parser;
+
+import ginger.command.*;
```

Comment:
Would it be clearer if you imported each class individually rather than used a wildcard import statement?